health-services/egov-notification-push/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM egovio/apline-jre:8u121
+
+MAINTAINER eGov Foundation
```

Hadolint (2.14.0)

[error] MAINTAINER is deprecated

(DL4000)